api/web.go

```diff
@@ -157,6 +157,17 @@ func (c *Config) HandleDownload(w http.ResponseWriter, r *http.Request) {
 	rand.Seed(time.Now().Unix())
 	host := c.Hosts[rand.Intn(len(c.Hosts))]
 	host = strings.Replace(host, "{{ preferred_username }}", userName, 1)
+	
+	
+	//Change Host to value of host query Parameter if host query string is in Config Hosts
+	paraHost, hasParaHost := r.URL.Query()["host"]
+	if hasParaHost {
+        for _,a := range c.Hosts {
+            if a==paraHost[0] {
```

Comment:
Please lint your code

Reply:
Lint hasn't suggested changes in new version